etc/omero.properties

```diff
@@ -196,6 +196,6 @@ versions.velocity=1.4
 ###
 
 ## Internal dependencies
-versions.omero-blitz=5.5.0-m9
-versions.omero-common-test=5.5.0-m8
-versions.omero-gateway=5.5.0-m6
+versions.omero-blitz=5.5.0
```

Comment:
Isn't that pulled transitively by `omero-gateway` ?

Reply:
Most of blitz should be excluded by gateway so both are needed.